shared/desktop/resolve-root.js

```diff
@@ -1,11 +1,18 @@
 // @flow
 import path from 'path'
-import electron from 'electron'
```

Comment:
had to hack around these things pulling in electron, which pulls in fs, which does stuff as soon as you import the module. this stuff can go away as we clean up these components